app/utils/deviceInfo.js

```diff
@@ -3,7 +3,15 @@ import DeviceInfo from 'react-native-device-info';
 
 import { MIN_WIDTH_SPLIT_LAYOUT } from '../constants/tablet';
 
-const NOTCH_DEVICES = ['iPhone X', 'iPhone XS', 'iPhone XS Max', 'iPhone XR'];
+const NOTCH_DEVICES = [
```

Comment:
Hey @t2hieu91, try to use [this function](https://github.com/react-native-community/react-native-device-info/tree/master#hasnotch) from `react-native-device-info` instead add more devices here. 😄 
Thanks in advance.